src/Components/Radio/Radio.component.jsx

```diff
@@ -77,14 +78,29 @@ const Radio = ({activeTrack, setActiveTrack}) => {
   const [status, setStatus] = useState(0);
   const [audio, setAudio] = useState(null);
   const [playing, setPlaying] = useState(false);
-  let nowPlaying = '<Song Name>';
-
-  if (activeTrack) {
-    nowPlaying = activeTrack.title;
-  }
+  //let nowPlaying = '<Song Name>';
+  const [nowPlaying, setNowPlaying] = useState('<CLICK PLAY!>');
+
+  const siteUrl = 'https://icecast.softboys.club:18000';
```

Comment:
nit

lets get a ticket on the board to host a global variable with our url, maybe the .env

Reply:
i was desperate to figure out global variables. set a ticket 
https://trello.com/c/RJFmP8E9